vite.config.js

```diff
@@ -8,6 +8,6 @@ export default defineConfig({
     react(),
     tailwindcss()
   ],
-  base: '/webtrendmonitor_frontend/',
+  // base: '/webtrendmonitor_frontend/',
```

Comment:
uncomment this line. It will allow the deployment into github. @iamazizbohra

Reply:
Call server API's endpoints. Server APIs are deployed. @iamazizbohra

Reply:
https://website-hackathon-backend.vercel.app/ - server api base path @iamazizbohra

Reply:
https://website-hackathon-backend.vercel.app/api/publish-build?buildId=123456&sitemap=https://example.com/sitemap.xml

Try this get API for verify @iamazizbohra

Reply:
ok